### ISSUE TITLE
`@vue-storefront/cli` needs Node.js 16 (Gallium) that is so old it is not even Maintenance LTS anymore.

### ISSUE THREAD

**Issue by kindaro** [DivanteLtd/vue-storefront#7014]
### Environment

- Operating System: Linux

### Reproduction

1. Make sure your version of Node.js is newer than 16 (Gallium).
2. Run `npx @vue-storefront/cli init` as offered in [official documentation](https://docs.vuestorefront.io/general/starting-new-project).
3. Write any name and select any integration template when prompted.
4. Observe error.

### Describe the bug

[You can look at the official Node.js release schedule here.](https://nodejs.org/en/about/previous-releases) As you see, Node.js 16 (Gallium) is not even Maintenance LTS anymore. This has various negative implications: this version may not be available in major distributions, it will not receive critical security updates, and so on.

I am not sure how much of Vue Storefront code base is affected. My journey with it is only at its outset. If it is only `cli` that is affected, then this issue is merely a nuisance, since I shall only need to run it a few times. But to run an outdated version of Node.js in production is irresponsible.

It is sad to see that an issue that seems to be related has been opened more than a year ago and even a pull request has been offered, without even a comment from the maintainers. I hate dealing with versioning issues as much as the next software engineer, but something must be done.

### Additional context

The issue #6844 seems to be related.

### Logs

```shell-script
% npx @vue-storefront/cli init
┌  Welcome to Vue Storefront 2 CLI! 💚
│
◇  🚀  What's your project name?
│  Example Project
│
◇  ⚙️  Choose an integration template:
│  Prestashop
│
◇  😩 Node.js version is incompatible. Please make sure you use Node.js version >= 16.13 and <17.
```
```

**Comment by lisaschumann** [comment]
Experiencing the same issue and surprised there has not been any update. It would be amazing, if someone took a look!

**Comment by skirianov** [comment]
hey! Thanks for raising this @kindaro, I'll take a look and try to roll out a fix asap.

**Comment by Kiwifu** [comment]
The same issue here, it seems that storefront is stuck on 16.13 and <17 node version, it's not possible to use it as I have a higher version of node. Please, we need help here, make Vue storefront compatible with the latest version of node.js/

**Comment by Kiwifu** [comment]
npx @vue-storefront/cli init
┌  Welcome to Vue Storefront 2 CLI! 💚
│
◆  🚀  What's your project name?
│  _
◇  🚀  What's your project name?
│  pwa
│
◇  ⚙️  Choose an integration template:
│  Magento 2
│
◇  😩 Node.js version is incompatible. Please make sure you use Node.js version >= 16.13 and <17.

I have the version Node.js 20, what to do?

**Comment by thomasdefilippis** [comment]
Has any progress been made on this issue? The use of node 16 implies that this repo is going stale or will not be maintained into the future.

**Comment by OliverRC** [comment]
Not to meantion an old version of Yarn which no matter what I try it refuses to recognize the yarn version 2.
Keeps moaning about 

```
Yarn version is incompatible. Please make sure you use Yarn version >= 1 and <3.
```

However when checking the version I've switched to version 2 of yarn

```
yarn -v 
# comes back with "2.4.3"
```

**Comment by andrea-merli** [comment]
I agree, the policies of my company would not even allow to use a version so outdated

**Comment by andrea-merli** [comment]
> Yeah,
> 
> I would guess that they are going to make community edition deprecated and move everything to enterprise. I decided to not use this project and instead use [medusajs ](https://medusajs.com/) which has worked out great for my team.

I suspect the same

**Comment by Tofandel** [comment]
I also think the same given they have now made private their Nuxt3 repo, and now everything is behind an "Information wall"

Shame because they started out great with opensource

**Comment by thomasdefilippis** [comment]
Yeah, it was a cool concept. The main reason that I was interested was the ability to have a single sign across multiple micro services of which the store is one. However, I discovered that is very easy to completely override authentication in wordpress, drupal, medusajs, and many other store and cms applications. Therefore, you can achieve a similar setup without shelling lots of cash. I received a quote from alokai after filling out their contact form for the enterprise demo. I won't share in order to be nice, but it was too high.

**Comment by Tofandel** [comment]
After a lot of searching I found the Nuxt 3 boilerplate repo has been moved to another org and is now for plentymarket
https://github.com/plentymarkets/plentyshop-pwa

I had a fork from a while back when it was still without any integration 
https://github.com/Tofandel/storefront-nuxt3-boilerplate
